hardhat.config.ts

```diff
@@ -1,36 +1,35 @@
 import { HardhatUserConfig } from "hardhat/config";
 import "@nomicfoundation/hardhat-toolbox";
-import "@nomiclabs/hardhat-etherscan";
 import "hardhat-contract-sizer";
 import { config as dotenv } from "dotenv";
 
 dotenv();
 
 const config: HardhatUserConfig = {
-  solidity: "0.8.17",
-  networks: {
-    localhost: {
-      url: "http://127.0.0.1:8545/",
+    solidity: {
+        compilers: [
+            {
+                version: "0.8.17",
+                settings: {
+                    optimizer: { enabled: true, runs: 200 },
+                    evmVersion: "istanbul",
```

Comment:
no need to mention the evmVersion explicitly. The default and the latest one is `london`